client/src/App.tsx

```diff
@@ -18,14 +19,26 @@ function App(): JSX.Element {
           <AuthProvider>
             <SocketProvider>
               <Switch>
-                <Route exact path="/login" component={Login} />
-                <Route exact path="/signup" component={Signup} />
-                <Route exact path="/dashboard">
+                {/* <Route path="/login">
+                  <Login />
+                </Route>
+                <Route path="/signup">
+                  <Signup />
+                </Route>
+                <Route path="/dashboard">
                   <Dashboard />
+                </Route> */}
+                <Route path="/login">
+                  <ProfileSettings />
+                  <Route path="/editProfile" />
```

Comment:
```suggestion
                  <Route path="/edit-profile" />
```
Please use hyphenated case for route names in JS. Ditto throughout!